reference/datetime/functions/mktime.xml

```diff
@@ -1,24 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- $Revision$ -->
-<!-- EN-Revision: bce4e5f7d7979c5b672e9375a17860b0ced5eef3 Maintainer: seros Status: ready -->
-<!-- Reviewed: no -->
```

Comment:
Please add the correct revcheck ID, as this is used to determine if a file is outdated or not:  https://doc.php.net/revcheck.php?p=files&lang=es

Which seems like it should be `349e3c6502e0bbeb15aef2b4a4a25f3f5e10fbfe`: https://doc.php.net/revcheck.php?p=plain&lang=es&hbp=bce4e5f7d7979c5b672e9375a17860b0ced5eef3&f=reference/datetime/functions/mktime.xml&c=on